<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>883</width>
    <height>752</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lbl_COMPort">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>COM Port</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cBox_COMPort">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pBtn1_Read">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>80</height>
         </size>
        </property>
        <property name="text">
         <string>Read data</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pBtn2_Get">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>80</height>
         </size>
        </property>
        <property name="text">
         <string>Get data</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pBtn3_Show">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>80</height>
         </size>
        </property>
        <property name="text">
         <string>Show data</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="hLayout"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
